### PR TITLE
protection and freedoms of pregnancy

### PR DESCRIPTION
**Pull request by WilliamMaber**
as we were discussing it on PPUK discord. I decided to write up a policy. this makes some changes to the policy around abortion. gives additional protections to foetuses with known disabilities. additionally states that the pirate party wants to protect against changes in the laws of abortions and the Freedom that women's get from that.
@Topperfalkon checkmate

### CODE REVIEW

**Comment by WilliamMaber** [comment]
I click on the wrong button. now it looks  like 
Screenshot from 2019-09-22 23-12-11
can someone fix if needed

**Comment by Topperfalkon** [comment]
That's fine, it just means it's not a draft. It needs to be reviewed to be submitted

File: Health/protection and freedoms of pregnancy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+protection and freedoms of pregnancy
+=============================================
+will protect the right to abortion as it stated under  Abortion Act 1967 of  England and Wales and Scotland.
+
+this includes the following clauses of the Act.
```
Comment (Topperfalkon) [inline Health/protection and freedoms of pregnancy.md]: I don't think we need to quote the bits we're keeping

Comment (Topperfalkon) [reply Health/protection and freedoms of pregnancy.md]: Linking to the legislation should be sufficient

File: Health/protection and freedoms of pregnancy.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* the termination is necessary to prevent grave permanent injury to the physical or mental health of the pregnant woman;
+
+* continuing the pregnancy would involve risk to the life of the pregnant woman (greater than if the pregnancy were terminated);
+
+the pirate party will change the law to repeal the following from the Abortion Act 1967.
+
+* there is a substantial risk that if the child were born, it would suffer from such physical or mental abnormalities as to be seriously handicapped.
+
+
+this is done to stop the possibility of a future of genetic based eugenics. especially in cases of prenatal test to see if they have Autism, dyspraxia,  dyslexia, Aspergers or other disabilities. it should be noted some of these disabilities are commonly used in large businesses today especially tech businesses.
```
Comment (Topperfalkon) [inline Health/protection and freedoms of pregnancy.md]: I accept this premise but question the ethics of allowing foetuses with severe deformities or other physical condition (like Huntingdon's, for example) which are known to significantly reduce lifespan and cause significant suffering.

**Pull request closed without merging**